src/main/java/com/synopsys/integration/alert/channel/jira/cloud/actions/JiraCloudGlobalTestAction.java

```diff
@@ -76,6 +79,16 @@ protected boolean isUserMissing(FieldAccessor fieldAccessor) throws IntegrationE
                    .noneMatch(email -> email.equals(username));
     }
 
+    @Override
+    protected boolean isUserAdmin(FieldAccessor fieldAccessor) throws IntegrationException {
+        JiraCloudProperties jiraProperties = createJiraProperties(fieldAccessor);
+        JiraCloudServiceFactory jiraCloudServiceFactory = jiraProperties.createJiraServicesCloudFactory(logger, gson);
+        MyPermissionsService myPermissionsService = jiraCloudServiceFactory.createMyPermissionsService();
+        MultiPermissionResponseModel myPermissions = myPermissionsService.getMyPermissions(JiraGlobalTestAction.JIRA_ADMIN_PERMISSION_NAME);
+        PermissionModel adminPermission = myPermissions.extractPermission(JiraGlobalTestAction.JIRA_ADMIN_PERMISSION_NAME);
+        return null != adminPermission && adminPermission.getHavePermission();
+    }
+
```

Comment:
This chunk is the actual feature. The rest is resolving the dependency hell from blackduck-common.